### ISSUE TITLE
When using horizontal cursor, the animation doesn't start at frame 0

### ISSUE THREAD

**Issue by ghost** [LottieFiles/lottie-interactivity#13]
Using the horizontal scroll functionality, when the cursor moves from left to right the animation seems to start from the middle frame, and not frame 0. It can be seen using this example:

https://lottiefiles.com/interactivity?url=https%3A%2F%2Fassets7.lottiefiles.com%2Fpackages%2Flf20_ltf45w.json#ninth

If you add controls to the lottie-player, you can clearly see the animation doesn't start from the beginning. I don't think this is a problem due to this specific animation as i've noticed it using others too, so I think its a player problem.

```
LottieInteractivity.create({
  player: '#zipper',
  mode: 'cursor',
  actions: [
    {
      position: { x: [0, 1], y: [-1, 2] },
      type: 'seek',
      frames: [0, 60],
    },
    {
      position: { x: -1, y: -1 },
      type: 'stop',
      frames: [0],
    },
  ],
});
```

**Comment by BoDonkey** [comment]
I notice I have this problem when I have the end frame number wrong. It goes through the first few frames so fast it seems like it starts in the middle.

**Comment by karamalie** [comment]
This issue is being worked on. Will update once completed

**Comment by karamalie** [comment]
Fixed on package "version": "0.1.4"

**Comment by ajayJayapalan** [comment]
Can anyone please tell me the meaning of  position y values. [ -1, 2].

Is position of x and y = -1 means outside the container ?
